sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -12,6 +12,46 @@
 
 ## 1.1.0 (2024-12-12)
 
+### Features Added
+- Added image support in content safety evaluators. Image URL or base64 encoded image can be passed in as part `conversation` input to the evaluators. See below for an example:
```

Comment:
Can you add all of the individual evaluator class names?

Reply:
Also add a note on all the multimodal evaluators that will be removed soon and recommend users to move to the core evaluators for all image/multimodal capabilties.